### PR TITLE
Fix docs for standalone flux pod

### PR DESCRIPTION
**Pull request by stephenmoloney**
What does this MR do?

- Minor doc fix where if the doc is for standalone, then
the helm operator should not be started

Why is this MR needed?

- To avoid confusion

### CODE REVIEW

**Comment by squaremo** [comment]
That _is_ a little confusing. If you're using helm to install flux, then we might presume you want the helm operator to run, too. But, this is in instructions for running flux, as you say ...

**Comment by stephenmoloney** [comment]
Agreed - which is why I am wondering if we should move this out of `site/standalone` completely and then move it into `site/helm` instead?

**Comment by dholbach** [comment]
Yes. I agreed, it'd make sense to move it under `site/helm`. Maybe into its `README.md`?

**Comment by stephenmoloney** [comment]
This is the current structure of helm readme:

```
./site/helm
|-- get-started.md
|-- helm-integration-requirements.md
`-- helm-integration.md
```

It doesn't fit in the latter 2, only `get-started.md` or a new file makes sense.

I could break up `get-started` into

1. For a public git repo do ... 

2. For a private git repo do...

sound ok ?

**Comment by dholbach** [comment]
The helm-integration*.md files are not very helpful right now. I filed #1205 about them.

I meant `chart/flux/README.md` earlier.

@squaremo What do you think about @stephenmoloney's suggestion?

**Comment by stephenmoloney** [comment]
I've made the suggested change in `site/helm/get_started.md` and removed from `site/standalone` so you can review and see what you think... 

https://github.com/stephenmoloney/flux/blob/b39a028e369f0485a40658939388b68a77153408/site/helm/get-started.md

**Comment by dholbach** [comment]
My idea of `get-started.md` to have a self-contained guide which takes you along well-defined steps to having flux up and running in front of you, kind of like a hello-world example.

Maybe document in the chart's README?

**Comment by squaremo** [comment]
> @squaremo What do you think about @stephenmoloney's suggestion?

It's reasonable to mention how to use known_hosts in a prominent place. The `get-started.md` file assumes you're using github, though, so it doesn't apply there (though it would be OK to have a warning, perhaps).

What we really need is more detailed and general instructions for installing it all with helm, but that would be rather unfairly expanding the scope of this PR :-) So let's just clear up the confusion, and separate Helm instructions from vanilla flux instructions.

File: site/helm/get-started.md [hunk 1]
```diff
@@ -59,23 +59,42 @@ Add the Flux repository of Weaveworks:
...
+      --set git.url=ssh://git@github.com/YOURUSER/flux-helm-test \
+      --set git.chartsPath=charts \
+      --namespace flux \
+      weaveworks/flux
+      ```
+      
+    - Using a private git repo:
+
+      ```sh
+      KNOWN_HOSTS='domain ssh-rsa line1
```
[the remark this answers is not in the document]
Comment (stephenmoloney) [reply site/helm/get-started.md]: I think these are valid points that should be addressed. 

### 1. How to get the value for known_hosts?

^ I'd like some advice for this part

### 2. How to pass the value for known_hosts into the `helm install` command

My own opinion is that we should show both ways

 1. Based on a `helm install --set-string known_hosts=${known_hosts}` method
 2. Based  on a `helm install --set-file known_hosts=path/to/file` method

The question then becomes, with this amount of information, where to put it without taking away from the flow of `getting_started.md` ...

**Comment by stephenmoloney** [comment]
```
My idea of get-started.md to have a self-contained guide which takes you along well-defined steps to having flux up and running in front of you, kind of like a hello-world example.
```
I can appreciate that - so an alternative option is to simply point a link to another `flux/chart/flux/README.md` with a little note `if using a private repo, take a look at flux/chart/flux/README.md`

And another reference again at the bottom of `getting_started.md` to `flux/chart/flux/README.md`.

If we agree this is a reasonable approach, then I can put that into a separate PR later and as @squaremo  said, just remove that documentation for now from `standalone/...`

**Comment by dholbach** [comment]
Sounds good to me.

Thanks a lot for working on this!

**Comment by stephenmoloney** [comment]
I've changed it again. So I'd like to hold off until I make another PR for `chart/flux/README.md` changes with details about the helm install. Will be a few days.

File: site/helm/get-started.md [hunk 2]
```diff
@@ -59,23 +59,34 @@ Add the Flux repository of Weaveworks:
...
+      --set git.url=ssh://git@github.com/YOURUSER/flux-helm-test \
+      --set git.chartsPath=charts \
+      --namespace flux \
+      weaveworks/flux
+      ```
+      
+    - Using a private git repo:
+       
+      When deploying from a private repo, the known_hosts of the git repo needs 
+      to be configured into a kubernetes secret so that `StrictHostKeyChecking` is respected.
```
Comment (squaremo) [inline site/helm/get-started.md]: It's a ConfigMap in the chart -- nitpick!

Comment (stephenmoloney) [reply site/helm/get-started.md]: yea, I'm writing the docs now for the README.md and I noticed that -- those are not exactly secret!

**Pull request merged**